spaceship/lib/spaceship/test_flight/base.rb

```diff
@@ -5,7 +5,12 @@ module Spaceship
   module TestFlight
     class Base < Spaceship::Base
       def self.client
-        @client ||= Client.client_with_authorization_from(Spaceship::Tunes.client)
+        # Initialize new client if none or team changed
+        if Spaceship::Tunes.client && (@client.nil? || @client.team_id != Spaceship::Tunes.client.team_id)
```

Comment:
What if `Spaceship::Tunes.client` is `nil`? Can it ever be `nil`? We handled it differently before this change I guess?

Reply:
I guess before this change it would have crashed?
```rb
@client ||= Client.client_with_authorization_from(Spaceship::Tunes.client)
```
Should we error out if its nil??

Reply:
I think we should error out, this would be a bug in fastlane if `@client` is nil by the end of this.